generate.lua

```diff
@@ -33,7 +33,8 @@ local args = lapp [[
 ]]
 
 local builder         = "lua" -- or "luajit"
-local doall           = args.release ~= "false" -- make doall a bool
+local doall           = args.release ~= "false" and args.name == "none" -- make doall a bool
+print("doall is " .. (doall and "true" or "false"))
```

Comment:
Right. But most changes in generate.lua will be undone again. Those were some first tests with individual class cominations, but I believe those are impractical.